communitymanager/src/main/java/org/pesmypetcare/communitymanager/managers/ChatManager.java

```diff
@@ -31,14 +35,21 @@
     /**
      * Default constructor.
      */
-    public ChatManager() {
-        db = FirebaseFirestore.getInstance();
+    public ChatManager(Context context) {
+        if (firebaseApp == null) {
+            FirebaseOptions.Builder builder = new FirebaseOptions.Builder()
+                    .setApplicationId(BuildConfig.FIREBASE_APPLICATION_ID).setApiKey(BuildConfig.API_KEY)
+                    .setDatabaseUrl(BuildConfig.DATABASE_URL).setStorageBucket(BuildConfig.STORAGE_BUCKET);
+            firebaseApp = FirebaseApp.initializeApp(context, builder.build());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5561031)